src/main/java/com/stripe/model/FeeRefundCollection.java

```diff
@@ -1,72 +1,97 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
 package com.stripe.model;
 
 import com.stripe.Stripe;
 import com.stripe.exception.StripeException;
 import com.stripe.net.ApiResource;
 import com.stripe.net.RequestOptions;
-
 import java.util.Map;
 
 public class FeeRefundCollection extends StripeCollection<FeeRefund> {
-  // <editor-fold desc="create">
   /**
-   * Create an application fee refund.
+   * Refunds an application fee that has previously been collected but not yet refunded. Funds will
+   * be refunded to the Stripe account from which the fee was originally collected.
+   *
+   * <p>You can optionally refund only part of an application fee. You can do so multiple times,
+   * until the entire fee has been refunded.
+   *
+   * <p>Once entirely refunded, an application fee can’t be refunded again. This method will raise
+   * an error when called on an already-refunded application fee, or when trying to refund more
+   * money than is left on an application fee.
    */
   public FeeRefund create(Map<String, Object> params) throws StripeException {
     return create(params, (RequestOptions) null);
   }
 
   /**
-   * Create an application fee refund.
+   * Refunds an application fee that has previously been collected but not yet refunded. Funds will
+   * be refunded to the Stripe account from which the fee was originally collected.
+   *
+   * <p>You can optionally refund only part of an application fee. You can do so multiple times,
+   * until the entire fee has been refunded.
+   *
+   * <p>Once entirely refunded, an application fee can’t be refunded again. This method will raise
+   * an error when called on an already-refunded application fee, or when trying to refund more
+   * money than is left on an application fee.
    */
   public FeeRefund create(Map<String, Object> params, RequestOptions options)
       throws StripeException {
-    return ApiResource.request(ApiResource.RequestMethod.POST, String.format("%s%s",
-        Stripe.getApiBase(), this.getUrl()), params, FeeRefund.class, options);
+    String url = String.format("%s%s", Stripe.getApiBase(), this.getUrl());
+    return ApiResource.request(
+        ApiResource.RequestMethod.POST, url, params, FeeRefund.class, options);
```

Comment:
2) Create (standard): Instance method to create sub-resource on collection already containing ID of the parent resource.

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/FeeRefundCollection.java#L38-L43